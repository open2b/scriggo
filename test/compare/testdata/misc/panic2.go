// paniccheck

package main

func main() {
	defer func() {
		defer func() {
			func() {
				defer func() {
					v := recover()
					println("recovered", v.(int))
				}()
				defer func() {
					panic(4)
				}()
				panic(3)
			}()
			recover()
			panic(5)
		}()
		panic(2)
	}()
	panic(1)
}
