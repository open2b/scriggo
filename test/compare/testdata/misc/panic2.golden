recovered 4
panic: 1
	panic: 2 [recovered]
	panic: 5